schema/artifacts.hcl

```diff
@@ -5,6 +5,11 @@ table "artifacts" {
     type    = uuid
     default = sql("generate_ulid()")
   }
+  column "agent_id" {
```

Comment:
do we need agent id in here? Since artifacts would just be linked to a check/playbook action that already contains the agent id?

Reply:
👍🏽